appcoins_net/src/main/java/com/appcoins/net/AppCoinsClient.java

```diff
@@ -0,0 +1,47 @@
+package com.appcoins.net;
+
+public class AppCoinsClient implements AppCoinsConnection {
```

Comment:
If this class has such relation with the ads model, does it make sense to in a different module? Its purpose doesn't exist out of the ads context.